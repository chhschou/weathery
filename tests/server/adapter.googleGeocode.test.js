import test from 'ava'

const { getAddrComponents } = require('../data/googleGeocode')

test('getAddrComponents works', t => {
  const expected = [
    {
      "level": 2,
      "longName": "Wellington City",
      "shortName": "Wellington City"
    },
    {
      "level": 1,
      "longName": "Wellington",
      "shortName": "Wellington"
    },
    {
      "level": 0,
      "longName": "New Zealand",
      "shortName": "NZ"
    }
  ]


  const actual = getAddrComponents()
  t.deepEqual(actual, expected)
})