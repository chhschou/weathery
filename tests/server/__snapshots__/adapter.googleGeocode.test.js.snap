// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extractLocation works 1`] = `
Object {
  "addrComponents": Array [
    Object {
      "level": 0,
      "longName": "New Zealand",
      "shortName": "NZ",
    },
    Object {
      "level": 1,
      "longName": "Wellington",
      "shortName": "Wellington",
    },
    Object {
      "level": 2,
      "longName": "Wellington City",
      "shortName": "Wellington City",
    },
  ],
  "bounds": Object {
    "northeast": Object {
      "lat": -41.2779306,
      "lng": 174.780461,
    },
    "southwest": Object {
      "lat": -41.2911216,
      "lng": 174.7707681,
    },
  },
  "coords": Object {
    "lat": -41.2900524,
    "lng": 174.7801039,
  },
}
`;
